ocs_ci/ocs/ui/storageclass.py

```diff
@@ -82,12 +92,20 @@ def delete_rbd_storage_class(self, sc_name):
 
         self.navigate_overview_page()
         self.navigate_storageclasses_page()
-        self.page_has_loaded()
+        self.page_has_loaded(retries=10)
         logger.info(f"sc_name is {sc_name}")
         self.do_click((f"{sc_name}", By.LINK_TEXT))
         self.do_click(self.sc_loc["action_inside_storageclass"])
         self.do_click(self.sc_loc["delete_inside_storageclass"])
         self.do_click(self.sc_loc["confirm_delete_inside_storageclass"])
         # wait for storageclass to be deleted
         time.sleep(2)
-        return not self.verify_storageclass_existence(sc_name)
+        if self.ocp_version_full >= version.VERSION_4_9:
+            self.page_has_loaded()
+            time.sleep(2)
+            logger.info(f"Looking for the storage class {sc_name}")
+            return not self.wait_until_expected_text_is_found(
+                (f"a[data-test-id={sc_name}]", By.CSS_SELECTOR), sc_name, 5
```

Comment:
You can increase the timeout for test stability, 10+

Reply:
Have increased the timeout